.github/workflows/publish.yml

```diff
@@ -113,7 +110,7 @@ jobs:
         env:
           DL_LINUX: "esthri-${{ env.VERSION }}-x86_64-unknown-linux-musl.tar.gz"
           DL_MAC: "esthri-${{ env.VERSION }}-x86_64-apple-darwin.tar.gz"
-          DL_WIN: "esthri-${{ env.VERSION }}-x86_64-pc-windows-msvc.tar.gz"
+          DL_WIN: "" # Do not publish
```

Comment:
I think it makes sense to leave this blank rather than removing it, can reviewers think of problems associated with doing so? To me this boils down to the file will not be found either way, so future modifiers will have an easier time tracing windows usage

Reply:
👍